panel/models/tabulator.py

```diff
@@ -52,6 +52,21 @@ def __repr__(self):
             f'value={self.value}, old={self.old})'
         )
 
+class SelectionEvent(ModelEvent):
+
+    event_name = 'selection-change'
+
+    def __init__(self, model, indices, selected):
```

Comment:
I am not the biggest fan of the names I have chosen.  

Your suggestion could also be confusing since the indices are only updated ones and not a full list of selected/deselected indices.

@philippjfr, any preferences?

Reply:
No real preference, a comment to clarify is probably more important than the naming here since these events won't be user facing.